modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -5,6 +5,8 @@
     :class="{'apos-area-widget-wrapper--foreign': foreign}"
     :data-area-widget="widget._id"
     :data-area-label="widgetLabel"
+    :data-apos-widget-foreign="foreign ? 1 : 0"
```

Comment:
Selecting an element with "0" as a value is easier than selecting an element with no value at all.